remotes/handlers.go

```diff
@@ -225,13 +228,14 @@ func PushContent(ctx context.Context, pusher Pusher, desc ocispec.Descriptor, st
 
 	platformFilterhandler := images.FilterPlatforms(images.ChildrenHandler(store), platform)
 
-	annotateHandler := annotateDistributionSourceHandler(platformFilterhandler, store)
+	var handler images.Handler
```

Comment:
This could be cleaner if it just overwrote the same handler variable and `var handler images.handler` only needs a single path like before.